src/software/new_geom/polynomial.h

```diff
@@ -45,7 +45,7 @@ class Polynomial
      * @param order the order of the term to set the coefficient
      * @param coeff the coefficient
      */
-    void setCoeff(unsigned int order, double coeff);
+    virtual void setCoeff(unsigned int order, double coeff);
```

Comment:
I think you can revert this change since line doesn't inherit polynomial

Reply:
Fixed in 23c9b47